project/scripts/release

```diff
@@ -298,10 +298,11 @@ if [ ! $dry_run ]; then
 else
   RELEASE_OPT="-Dakka.genjavadoc.enabled=true"
 fi
-try sbt $RELEASE_OPT +buildRelease
-try sbt -Dakka.build.scalaVersion=2.11.12 -D$RELEASE_OPT buildRelease
-try sbt $RELEASE_OPT clean
-try sbt $RELEASE_OPT buildDocs
+# Needs to be kept in sync with project/Dependencies.scala manually
+try sbt -Dakka.build.scalaVersion=2.12.8 $RELEASE_OPT clean buildRelease
+try sbt -Dakka.build.scalaVersion=2.13.0 $RELEASE_OPT clean buildRelease
+try sbt -Dakka.build.scalaVersion=2.11.12 $RELEASE_OPT clean buildRelease
+try sbt $RELEASE_OPT clean buildDocs
```

Comment:
just to cut down on the build time we could maybe build 2.12 last and then not clean before buildDocs? (but not important)